app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from typing import List
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(
+        players: List[Player]
+) -> int:
+    return sum(
+        player.get_rating()
+        for player in players
+    )
+
+
+def elves_concert(elfs: List[Elf]) -> None:
```

Comment:
Comprehension means something should be created, use a for loop for actions to be called

Reply:
> Comprehension means something should be created, use a for loop for actions to be called

Yeah, but it looks so beautiful with comprehension)))